### PR TITLE
Skip empty epoch

### PR DESCRIPTION
**Pull request by kstdl**

### CODE REVIEW

File: src/connectors/TokenState.sol [hunk 1]
```diff
@@ -5,33 +5,34 @@ pragma solidity ^0.8.17;
...
 }
 
-contract TokenState {
+contract TokenState is Ownable {
     uint256 public epoch;
     address[] accounts;
     mapping(address => uint256) balances; // position = 2
 
-    constructor(uint256 _epoch) {
+    constructor(uint256 _epoch) Ownable() {
```
Comment (VargaElod23) [inline src/connectors/TokenState.sol:20]: Who is the owner here? This contract is being deployed by the bridge contract but let's make sure that the ownership isn't on the caller.

**Review by VargaElod23 — CHANGES_REQUESTED** [review]
Pls review the comment then can be accepted

File: src/connectors/TokenState.sol (hunk 1 above)
Comment (kstdl) [reply src/connectors/TokenState.sol:20]: This contract is created(deployed?) by the Connector contract. So this Connector will be an Owner

**Review by VargaElod23 — APPROVED** [review]
LGTM

**Pull request merged**